packages/video_player/example/lib/main.dart

```diff
@@ -373,6 +377,20 @@ void main() {
           ),
           body: new TabBarView(
             children: <Widget>[
+              new Column(
+                children: <Widget>[
+                  new NetworkPlayerLifeCycle(
+                    'http://www.sample-videos.com/video/mp4/720/big_buck_bunny_720p_20mb.mp4',
+                    (BuildContext context, VideoPlayerController controller) =>
+                        new AspectRatioVideo(controller),
+                  ),
+                  new NetworkPlayerLifeCycle(
+                    'http://live.cloud6.in:1935/simskasri/limitless/playlist.m3u8',
```

Comment:
I think we need something less controversial here. I don't know this feed, but the first thing I got when I tried it was a preacher.

Reply:
Sure. I've changed it to a big buck bunny hls stream.

Reply:
It's hard to control the top video without interfering with the bottom one. They seem to be too close. Can I ask you to add some padding?